kmapper/kmapper.py

```diff
@@ -615,6 +615,7 @@ def visualize(
         self,
         graph,
         color_function=None,
+        colorscale=colorscale_default,
```

Comment:
default to the colorscale defined in visuals. this is where we could potentially do `colorscale=_return_default_colorscale()` if we didn't want a mutable default value for a kwarg

Reply:
I would definitely prefer the function return :D. The mutable default values was a bit of a sticking point during our JOSS submission.